pylint/test/unittest_checker_variables.py

```diff
@@ -89,6 +90,20 @@ def test():
             self.checker.visit_module(node.root())
             self.checker.visit_functiondef(node)
 
+    def test_unassigned_global(self):
+        node = astroid.extract_node('''
```

Comment:
You can simplify this a lot if you use the following:
```
node = extract_node('''
def func():
   global sys #@
   import sys
'''')
```
What happens with that `# @` is that you specify which node to extract from a tree so it should be enough!